Lib/codeop.py

```diff
@@ -66,7 +66,10 @@ def _maybe_compile(compiler, source, filename, symbol):
                 compiler(source + "\n", filename, symbol)
                 return None
             except SyntaxError as e:
-                if "incomplete input" in str(e):
+                strerr = str(e)
```

Comment:
```suggestion
                # XXX: RustPython; support multiline definitions in REPL
                # See also: https://github.com/RustPython/RustPython/pull/5743
                strerr = str(e)
```
Because this is a part of standard library from CPython, leaving comment helps not to let them unintentionally deleted by udpate